sources/store.c

```diff
@@ -421,6 +422,9 @@ int CoSave(UBYTE *inp)
 			c = *p; *p = 0;
 			if ( GetVar(inp,&type,&number,CEXPRESSION,NOAUTO) != NAMENOTFOUND ) {
 				if ( e[number].status == STOREDEXPRESSION ) {
+					if (StrLen(AC.exprnames->namebuffer+e[number].name) > MAXENAME) {
```

Comment:
Maybe you can apply the same coding style as other if-statements (putting a space in parentheses inside conditional statements). The same for the other line with a similar if-statement.

Reply:
Right, this is not the first time I forgot this one...